assets/app/styles/_core.less

```diff
@@ -404,6 +413,14 @@ code {
   white-space: normal;
 }
 
+pre {
+  &.code {
```

Comment:
This is a code block with multiple lines. I understand the `<code>` tag is meant to be used for inline code, and the `<pre>` tag from Patternfly has quite simple styling, pretty much it adds a border (my first screenshot shows it). 

We wanted the code block to pop out more, Patternfly suggests you use [prettify](https://www.patternfly.org/widgets/#google-code-prettify) but since this is just bash, you don't get much from that. 

I don't see us using `<pre>` for things other than multi-line code blocks, so I'd say this could be a suggested style change.